modules/mobile-agents/windows/jax-rs/src/main/java/org/wso2/carbon/mdm/mobileservices/windows/operations/Replace.java

```diff
@@ -30,42 +29,42 @@
  * Commands sent from the device.
  */
 public class Replace {
```

Comment:
Using a Verb for a Class is inappropriate. Better to use something like TagReplace